salt/modules/zypper.py

```diff
@@ -555,7 +555,7 @@ def info_available(*names, **kwargs):
         if nfo.get('status'):
             nfo['status'] = nfo.get('status')
         if nfo.get('installed'):
-            nfo['installed'] = nfo.get('installed').lower() == 'yes' and True or False
+            nfo['installed'] = nfo.get('installed').lower().startswith( 'yes' ) and True or False
```

Comment:
Isn't the half of this statement after the `and` now redundant?

Reply:
Yes, `startswith()` already returns a bool.